src/Core/src/Core/IWindow.cs

```diff
@@ -134,5 +134,9 @@ public interface IWindow : ITitledElement
 		ITitleBar? TitleBar => null;
 		Rect[]? TitleBarDragRectangles => null;
 #endif
+
+#if MACCATALYST
```

Comment:
Can simplify this. ITitlebar is on Windows and Catalyst. Just need:

```
     ITitleBar? TitleBar => null;

#if WINDOWS
     Rect[]? TitleBarDragRectangles => null;
#endif
```

Reply:
So something like this?

```
#if WINDOWS || MACCATALYST
		ITitleBar? TitleBar => null;
#endif

#if WINDOWS
		Rect[]? TitleBarDragRectangles => null;
#endif
```